app/components/views/TicketsPage/PurchaseTab/LEGACY_PurchasePage/LEGACY_TicketAutoBuyer/index.js

```diff
@@ -151,11 +151,7 @@ class TicketAutoBuyer extends React.Component {
   }
 
   getIsFormValid() {
-    return (
-      this.getAccount() &&
-      this.state.balanceToMaintain > 0 &&
-      this.getStakePool()
-    );
+    return this.getAccount() && this.getStakePool();
```

Comment:
maybe check if balance to mantain is `>=0`?

Reply:
Sure. I've added it.